components/screens/LoginScreen.js

```diff
@@ -0,0 +1,237 @@
+} from 'react-native';
+
+import { LinearTextGradient } from 'react-native-text-gradient';
+import LinearGradient from 'react-native-linear-gradient';
+
+import { useNavigation } from '@react-navigation/native';
+
+import Icon from "react-native-vector-icons/FontAwesome";
+import CustomButton from '../reusable/CustomButton';
+import CustomInput from '../reusable/CustomInput';
+
+function LoginScreen() {
+    const navigation = useNavigation()
+
+    const handleCreateNewAccount = () => {
+        navigation.navigate("RegisterScreen");
+    }
+
+    const handleLogIn = () => {
+        navigation.navigate("Dashboard")
+    }
+    return (
+        <View>
+            <ImageBackground
+                resizeMode={"cover"}
+                source={require("../../assets/images/loginBg.png")}
+            >
+                <LinearGradient
+                    colors={[
+                        "rgba(255, 255, 255, 0)",
+                        "rgba(242, 242, 242, 1)"
+                    ]}
+                    locations={[0, 0.9]}
+                    style={{
+                        paddingVertical: 100
+                    }}
+                >
+                    <Image
+                        source={require("../../assets/images/logo.png")}
+                        style={{
+                            width: 100,
+                            height: 100,
+                            alignSelf: "center"
+                        }}
+                    />
+
+                    <LinearTextGradient
+                        style={{
+                            fontSize: 25,
+                            fontWeight: "700",
+                            alignSelf: "center"
+                        }}
+                        colors={[
+                            "#E54500CF",
+                            "#FF5C00C9",
+                            "#FF9921"
+                        ]}
+                        locations={[0, 0.5, 1]}
+                        start={{ x: 1, y: 0 }}
+                        end={{ x: 0, y: 0 }}
+                    >
+                        <Text>Spoon</Text>
+                    </LinearTextGradient>
+                </LinearGradient>
+            </ImageBackground>
+            <View style={styles.loginButtonsContainer}>
+                <TouchableOpacity style={styles.loginButton}>
+                    <Icon
+                        name="google"
+                        size={15}
+                        color="red"
+                        style={styles.loginButtonIcons}
+                    />
+                    <View style={styles.LoginButtonTextContainer}>
+                        <Text style={styles.loginButtonText}>
+                            Continue with Google
+                        </Text>
+                    </View>
+                </TouchableOpacity>
+                <TouchableOpacity style={styles.loginButton}>
+                    <Icon
+                        name="facebook"
```

Comment:
this should be reusable component you are using the same code for google and facebook